qiskit/ignis/mitigation/measurement/filters.py

```diff
@@ -208,22 +222,22 @@ def _apply_correction(self, resultidx, raw_data, method):
 
 class TensoredFilter():
     """
-    Tensored measurement error mitigation filter
+    Tensored measurement error mitigation filter.
 
     Produced from a tensored measurement calibration fitter and can be applied
     to data.
     """
 
-    def __init__(self, cal_matrices, substate_labels_list):
+    def __init__(self,
+                 cal_matrices: np.matrix,
+                 substate_labels_list: list):
```

Comment:
The list type does not appear well in the doc.

Reply:
fixed